frontend/src/index.jade

```diff
@@ -60,10 +60,10 @@ html
                   span &nbsp; icon on the left.
 
       template(v-else)
-        .empty please enter a report directory or upload a report zip
+        .empty Please upload a .zip file that is generated by RepoSense.
 
         form#file-upload(onsubmit="return false;")
-          input(type="file", v-on:change="updateReportZip")
+          input(id="fileUpload", type="file", accept=".zip", v-on:change="updateReportZip")
```

Comment:
why did you added in an `id` here, if not needed, can remove.

Reply:
No longer needed.  The ID was before I integrated the checks with the existing .zip file checker. Removed the ID in the latest commit.